session/pessimistic_test.go

```diff
@@ -370,7 +372,6 @@ func (s *testPessimisticSuite) TestOptimisticConflicts(c *C) {
 	tk.MustExec("update conflict set c = 4 where id = 1")
 	tk2.MustExec("begin optimistic")
 	tk2.MustExec("update conflict set c = 5 where id = 1")
-	// TODO: ResolveLock block until timeout, takes about 40s, makes CI slow!
```

Comment:
I'd rather delete the following line (skip the test), what do you think? @coocood